src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml

```diff
@@ -15,7 +15,7 @@
     </Design.DataContext>
     <Button Classes="SpineDownloadButton Idle" x:Name="ParentButton">
         <ToolTip.Tip>
-            <TextBlock x:Name="ToolTipTextBlock"/>
+            <TextBlock Classes="ToolTipText" x:Name="ToolTipTextBlock"/>
```

Comment:
Why is this class needed? If every `TextBlock` inside a `ToolTip` should have this class, then the class shouldn't exist in the first place, and all the styles of the class should be made the default.

Reply:
Tooltips should never contain anything other than text and maybe an icon. Putting a button in a tooltip is pure evil.

Reply:
For `ToolTip` specifically, just do `ToolTip TextBlock`.

Reply:
In MO2 we have tables in there, just saying